internal/service/docdb/cluster_instance.go

```diff
@@ -313,11 +327,13 @@ func resourceClusterInstanceRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("availability_zone", db.AvailabilityZone)
 	d.Set("cluster_identifier", db.DBClusterIdentifier)
 	d.Set("dbi_resource_id", db.DbiResourceId)
+	//d.Set("enable_performance_insights", db.EnablePerformanceInsights)
```

Comment:
Comment above that aws api does not expose these values and they should be uncommented if theyre ever added to `DescribeDBClusters` output